pkgs/development/libraries/tagparser/default.nix

```diff
@@ -1,29 +1,34 @@
 { stdenv
-, pkgs
+, lib
 , fetchFromGitHub
 , cmake
 , cpp-utilities
 , zlib
+, isocodes
 }:
 
 stdenv.mkDerivation rec {
   pname = "tagparser";
-  version = "9.4.0";
+  version = "11.5.0";
 
   src = fetchFromGitHub {
     owner = "Martchus";
     repo = "tagparser";
     rev = "v${version}";
-    sha256 = "097dq9di19d3mvnlrav3fm78gzjni5babswyv10xnrxfhnf14f6x";
+    hash = "sha256-qgopl32cFQFQTYu9WBOzPeU69J8K49SREX7X0Pw7Als=";
   };
 
   nativeBuildInputs = [ cmake ];
 
-  buildInputs = [
-    cpp-utilities zlib
-  ];
+  buildInputs = [ cpp-utilities zlib ];
 
-  meta = with pkgs.lib; {
+  postPatch = ''
+    substituteInPlace ./cmake/scripts/generate_iso_language_codes.cmake \
+    --replace "/usr/bin/cmake" "${cmake}/bin/cmake" \
+    --replace "/usr/share/iso-codes" "${isocodes}/share/iso-codes"
```

Comment:
Did you try using cmake flag:

```
  cmakeFlags = [
    "-DLANGUAGE_FILE_ISO_639_2=${isocodes}/share/iso-codes/json/iso_639-2.json"
  ];
```

See https://github.com/Martchus/tagparser/blob/93da1f1e25bde72a283e26fca6c7e436892645bb/CMakeLists.txt#L213

As for the cmake shebang, I'm not sure it needs patching, because the `CMakeLists.txt` already calls it with `${CMAKE_COMMAND}`:

https://github.com/Martchus/tagparser/blob/93da1f1e25bde72a283e26fca6c7e436892645bb/CMakeLists.txt#L219